.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
```

Comment:
this copies the linting from scripts/lint, but runs yapf in in-place correction mode to actually fix the problems that would be errored in CI